src/modules/verifiable-credentials/verifiable-credentials-node.service.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  prepareIssueCredential,
+  completeIssueCredential,
+  verifyCredential,
+  verifyPresentation,
+} from 'didkit-wasm-node';
+import {
+  prepareIssuePresentation,
+  completeIssuePresentation,
+} from '@spruceid/didkit-wasm-node';
+import { SignerService } from '../signer';
+import { VerifiableCredentialsServiceBase } from './verifiable-credentials-base.service';
+
+export class VerifiableCredentialsServiceNode extends VerifiableCredentialsServiceBase {
+  protected prepareIssueCredential: (
```

Comment:
why duplicate declarations in sub-classes?

Reply:
To have ability to use here the function from DIDKit `this.prepareIssueCredential = prepareIssueCredential`

Reply:
Isn't possible to override them without re-declaration like
```
prepareIssueCredential(...) {
   return prepareIssueCredential(...)
}
```

Reply:
It is possible. But why this is better then simple overriding?

Reply:
Overriding is happen in any case either as method or property overriding, but method overriding allows to avoid re-declaration

Reply:
I don't think so. It's extending an abstract method, so it have to be declared in derived class

Reply:
ok